packages/app/cypress/tasks/git.ts

```diff
@@ -30,3 +30,11 @@ export async function initGitRepoForTestProject (projectPath: string) {
 
   return null
 }
+
+export async function resetGitRepoForTestProject (projectPath: string) {
+  const git = simpleGit({ baseDir: projectPath })
+
+  await git.reset(['--hard'])
+
+  return null
```

Comment:
How come you're returning `null`? Is TS requiring you to have a return value?

Reply:
I guess I stole the pattern from the function above it. Upon digging up, it seems it is required to return something per the comment on the code shown below
https://github.com/cypress-io/cypress/blob/d6df1069dc934e4a017e20917231616f472cd0b3/cli/types/cypress.d.ts#L5472-L5477